models/multi_organ_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,91 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.0.2",
+    "changelog": {
+        "0.0.1": "initialize the model package structure",
+        "0.0.2": "Retrain with new MONAI"
+    },
+    "monai_version": "1.2.0rc7",
+    "pytorch_version": "2.0.0a0+1767026",
```

Comment:
Hi @cshen0625 , here could you use `"pytorch_version": "1.13.1"`? We may need a formal torch version (rather than a dev version).
In addition, I suggest to use `"monai_version": "1.2.0"`

Reply:
Sure. I will test if it works with 1.2.0.
I used the MONAI container with tag 1.2.0rc7, which was the latest version available at that time.

Reply:
Hi @yiheng-wang-nv , this bundle works with `"monai_version": "1.2.0"` and `"pytorch_version": "1.13.1"`. Can I just modify the information in `metadata.json`?

Reply:
Yes @cshen0625 . Thanks!

Reply:
Hi @yiheng-wang-nv , I updated the `metadata.json` file.